pkg/app/app.go

```diff
@@ -70,19 +71,19 @@ const (
 type App struct {
 	broker   *broker.AnsibleBroker
 	args     Args
-	config   Config
+	config   *config.Config
 	dao      *dao.Dao
 	log      *Log
 	registry []registries.Registry
 	engine   *broker.WorkEngine
 }
 
-func apiServer(log *logging.Logger, config Config, args Args, providers []auth.Provider) (*genericapiserver.GenericAPIServer, error) {
+func apiServer(log *logging.Logger, config *config.Config, args Args, providers []auth.Provider) (*genericapiserver.GenericAPIServer, error) {
 	log.Debug("calling NewSecureServingOptions")
 	secureServing := genericoptions.NewSecureServingOptions()
 	secureServing.ServerCert = genericoptions.GeneratableKeyCert{CertKey: genericoptions.CertKey{
-		CertFile: config.Broker.SSLCert,
-		KeyFile:  config.Broker.SSLCertKey,
+		CertFile: config.GetString("broker.ssl_cert"),
+		KeyFile:  config.GetString("broker.ssl_cert_key"),
```

Comment:
So `config.Broker.SSLCert` gave us the compile time benefit. But it was also the source of what made adding stuff to the config painful. I would have to go update the struct vs just adding values to the config. Most projects I've worked with I've used the `GetString` type methods.